spec/controllers/v0/sign_in_controller_spec.rb

```diff
@@ -757,14 +763,23 @@
 
     let(:user_verification) { create(:user_verification) }
     let(:user_verification_id) { user_verification.id }
+    let(:user) { create(:user) }
```

Comment:
Make this `let!(:user)` here and you don't need to mock the User find in that `before do` below